### ISSUE TITLE
project onboarding:  Project Infrastructure SIG

### ISSUE THREAD

**Issue by svrnm** [open-telemetry/community#2186]
With https://github.com/open-telemetry/community/pull/2096 merge, the Project Infrastructure SIG can be onboarded, here are the TODOs to get them started:

- [x] Create project board
- [x] SIG Slack Channel
- [x] Meeting Notes (Google Docs)
- [x] Add Meeting Times, GC Liaison, Slack Channel, Google Doc to sigs.yml
- [x] GitHub Teams
- [x] Repository (?)

cc @austinlparker @trask @jaronoff97

**Comment by svrnm** [comment]
Has the board https://github.com/orgs/open-telemetry/projects/91 been created for this SIG?

**Comment by svrnm** [comment]
Google Docs: https://docs.google.com/document/d/1FgwSYgNDOvdmbf7HqOYUodzDpfzPA4pDLPhgF8sK4F4/edit#heading=h.x3xl5syz15zx

**Comment by austinlparker** [comment]
Yes, https://github.com/orgs/open-telemetry/projects/91/views/1 is the board. https://docs.google.com/document/d/1_dTP2XIaZoFjNiipkeMaGJN10NrhS20FiBTTJoQJeVM/edit#heading=h.7dgdp4q4kklh is the meeting notes. 

I'm not sure if we'll need a separate repo yet to be honest.

**Comment by svrnm** [comment]
> I'm not sure if we'll need a separate repo yet to be honest.

If people do not feel spammed if we use the community forum for raising issues, we might not need one immediately. But I wonder if we need one eventually to host some of the tooling, e.g. if we create shared github actions, except we host them in a place like https://github.com/open-telemetry/.github

**Comment by austinlparker** [comment]
We've got teams and repos set up so this is done.

@open-telemetry/project-infra https://github.com/open-telemetry/sig-project-infra
